GLTFSerialization/GLTFSerialization/GLTFHelpers.cs

```diff
@@ -260,11 +261,21 @@ public static void BuildMeshAttributes(ref Dictionary<string, AttributeAccessor>
 						attributeAccessor.AccessorId.Value.AsUIntArray(ref resultArray, bufferViewCache, offset);
 						break;
 					default:
-						throw new System.Exception($"Unrecognized mesh attribute [{kvp.Key}]");
+						unrecognizedAttributes.Add(kvp.Key);
+						continue;
 				}
 				
 				attributeAccessor.AccessorContent = resultArray;
 			}
+
+			foreach (var attrib in unrecognizedAttributes) { 
+				attributes.Remove(attrib);
+			}
+
+			// TODO This should be a warning. Unrecognized attributes (e.g. TEXCOORD_4) should not cause full exceptions.
+			if (unrecognizedAttributes.Count > 0) {
+				throw new System.Exception($"Unrecognized mesh attributes [{string.Join(", ", unrecognizedAttributes.ToArray())}]");

```

Comment:
>System.Exception [](start = 14, length = 16)

Not exactly your change, but this should probably be a GLTFLoadException.

Reply:
Ah, good, I was looking for the right one but didn't find it on first glance. I'll change it, of course don't want to catch all.

Reply:
OK, think I misunderstood you but I now added a GLTFLoadException for this and future usecases.